examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -364,6 +413,13 @@ CHIP_ERROR CommandHandler(int argc, char ** argv)
                                     "Commissioner-Generated Passcode commissioning flow",
                                     index);
                     idOptions.mCommissionerPasscode = true;
+
+                    // Override the default target Vendor ID for this example (Commissioner-Generated passcode commissioning flow).
+                    // This tv-app target Vendor ID (1111), does not implement the AccountLogin cluster, which would otherwise auto
+                    // commission using the Commissionee-Generated passcode upon recieving the IdentificationDeclaration Message.
+                    // See connectedhomeip/examples/tv-app/tv-common/include/AppTv.h.
+                    targetAppInfo.vendorId             = 1111;
```

Comment:
Since this endpoint is not reporting its vendorId, why are we setting this targetAppInfo.vendorId?

Reply:
Even though the Linux tv-app is not reporting this vendorId we know it is valid since it is defined in connectedhomeip/examples/tv-app/tv-common/include/AppTv.h:

    ContentAppImpl mContentApps[APP_LIBRARY_SIZE] = {
        ContentAppImpl("Vendor1", 1, "exampleid", 11, "Version1", "34567890"),
        ContentAppImpl("Vendor2", 65521, "exampleString", 32768, "Version2", "20202021"),
        ContentAppImpl("Vendor3", 9050, "App3", 22, "Version3", "20202021"),
        ContentAppImpl("TestSuiteVendor", 1111, "applicationId", 22, "v2", "20202021")
    };

We need to use this endpoint to demo the Commissioner-Generated passcode flow since this endpoint doesn't have AccountLogin implemented. The default target tv-app with vendorId 65521 would otherwise auto-commission using AccountLogin and the hardcoded passcode "20202021".